docs/developer/app-store/apps/product-feed.mdx

```diff
@@ -0,0 +1,139 @@
+import { AppMetadata } from "/components/AppMetadata/AppMetadata.tsx";
+
+# Product feed
+
+## Metadata
+
+<AppMetadata
+  minSaleorVersion="3.10"
+  roadmapUrl="https://github.com/saleor/apps/labels/App%3A%20Product%20Feed"
+  githubUrl="https://github.com/saleor/apps/tree/main/apps/products-feed"
+/>
+
+## Introduction
+
+The app generates XML files in Google Merchant feed format.
+
+## Features
+
+- Generate feed in the [Google Merchant format](https://support.google.com/merchants/answer/7439882)
+- Map your catalog structure to Google Merchant format
+- Upload files to the AWS S3 bucket
+- Separate feed for each of the channels
+
+## Assumptions & limitations
+
+- Each item in the feed is equivalent to a single Product Variant in Saleor
+- For catalogs bigger than 200 product variants an AWS S3 bucket is recommended
+- Translations are not supported - product data are generated using the default language
+- Only the public data are used to generate the feed. It means that not published products will not be included
+- The feed is generated upon visiting the feed URL, which can be found in the app configuration
+- Once generated feed, it will be cached for one day. [Update your scheduled fetches accordingly](https://support.google.com/merchants/answer/7371671).
+- Data displayed in Google Merchant Center do not update in real-time. It may take [up to 3 business days](https://support.google.com/merchants/answer/188477) to update.
+
+## Glossary
+
+- [**Google Merchant Center**](https://www.google.com/intl/en_us/retail/solutions/merchant-center/): Service used to share product data in search results and ads
+- [**AWS S3 Bucket**](https://aws.amazon.com/s3/): File hosting service created by Amazon
+- [**Handlebars**](https://handlebarsjs.com/): Language used for creating templates. The app uses it for generating dynamic titles and URLs based on product data
+
+## Permissions
+
+The app requires `MANAGE_PRODUCTS` permission to be able to subscribe to webhooks triggered upon changes in your catalog.
+
+The staff user configuring the App must have `MANAGE_APPS` permission.
+
+## Application flow
+
+The app will subscribe for webhooks:
```

Comment:
The whole section can be reduced - the app no longer listens to the webhooks.

The feed is generated on request time and uploaded to the s3 is configured.